signing/src/main/java/tech/pegasys/web3signer/signing/config/SecpArtifactSignerProviderAdapter.java

```diff
@@ -55,8 +55,8 @@ public Future<Void> load() {
   }
 
   @Override
-  public Optional<ArtifactSigner> getSigner(String identifier) {
-    return Optional.ofNullable(signers.get(identifier));
+  public Optional<ArtifactSigner> getSigner(String address) {
```

Comment:
nit: eth1Address may be even better

Reply:
my first change was eth1Address but then I thought that eth1 would be sort of implicit when it comes to secp, I could be wrong though